cljfmt/src/cljfmt/config.clj

```diff
@@ -59,6 +59,48 @@
 (defn- directory? [path]
   (some-> path io/file .getAbsoluteFile .isDirectory))
 
+(defn- deep-merge
+  "Recursively merges maps together. If all the maps supplied have nested maps
+  under the same keys, these nested maps are merged. Otherwise the value is
+  overwritten, as in `clojure.core/merge`."
+  {:arglists '([& maps])
+   :added    "1.1.0"}
+  ([])
+  ([a] a)
+  ([a b]
+   (when (or a b)
+     (letfn [(merge-entry [m e]
+               (let [k  (key e)
+                     v' (val e)]
+                 (if (contains? m k)
+                   (assoc m k (let [v (get m k)]
+                                (if (and (map? v) (map? v'))
+                                  (deep-merge v v')
+                                  v')))
+                   (assoc m k v'))))]
+       (reduce merge-entry (or a {}) (seq b)))))
+  ([a b & more]
+   (reduce deep-merge (or a {}) (cons b more))))
+
+(defn- load-extra-configs
+  "Load any :extra-configs specified in the base config file.
+  Notes:
+    * Not recursive.
+    * Only select keys are merged (`:extra-indents`)."
```

Comment:
Docstrings for private functions should be omitted.

Reply:
Couple of ways I could interpret this feedback:

* Either that's an argument that the code is mistructured and this is a code smell that I thought it was worthy of a comment, and should be refactored (self-documenting, or extracted from ns, or some other thing?)
* docstrings/comments aren't helpful or are confusing in private fn's? or perhaps is messes with some sort of automation or something?

I'm unsure how to accept the latter. I can just yank docstrings from private fn's I guess? But this feedback feels like shorthand for the former I'd assume?

Reply:
I just removed the docstr w/ the rewrite to change it to be `merge-extra-configs`, let me know if there's some other thing you want done